src/test/resources/pages/instructorFeedbackEmptyAll.html

```diff
@@ -1388,4 +1388,4 @@ <h4 class="modal-title">
     </div>
 </div>
 
-<div class="ui-datepicker ui-widget ui-widget-content ui-helper-clearfix ui-corner-all" id="ui-datepicker-div"></div></body></html>
\ No newline at end of file
+<div class="ui-datepicker ui-widget ui-widget-content ui-helper-clearfix ui-corner-all" id="ui-datepicker-div"></div><div class="tooltip fade top in" style="top: {*}px; left: {*}px; display: block;"><div class="tooltip-arrow"></div><div class="tooltip-inner">Enter the name of the feedback session e.g. Feedback Session 1.</div></div></body></html>
```

Comment:
Yeah, perhaps we can open issues to deal with those "imperfections" one of these days :D

If we don't deal with them eventually, soon enough we might have it go pass code review and then things might break :\